### PR TITLE
chore: some throughput tweaks

### PR DESCRIPTION
**Pull request by tkporter**
### Description

Context: https://discord.com/channels/935678348330434570/1254036693217054794

### CODE REVIEW

File: rust/Cargo.lock [hunk 1]
```diff
@@ -7037,6 +7037,7 @@ dependencies = [
  "num-traits",
  "once_cell",
  "prometheus",
+ "rand 0.5.6",
```
Comment (paulbalaji) [inline rust/Cargo.lock:7040]: why is the version here different to the one in the Cargo.toml?

File: typescript/infra/src/agents/index.ts [hunk 2]
```diff
@@ -139,7 +139,7 @@ export abstract class AgentHelmManager {
             rpcConsensusType: this.rpcConsensusType(chain),
             protocol: metadata.protocol,
             blocks: { reorgPeriod },
-            maxBatchSize: 4,
+            maxBatchSize: 16,
```
Comment (daniel-savu) [inline typescript/infra/src/agents/index.ts:142]: are we bumping this to 32 everywhere following our conversation earlier?

File: typescript/infra/config/environments/mainnet3/agent.ts [hunk 3]
```diff
@@ -285,9 +285,9 @@ const releaseCandidate: RootAgentConfig = {
     rpcConsensusType: RpcConsensusType.Fallback,
     docker: {
       repo,
-      tag: '0d12ff3-20240620-173353',
+      tag: '9535087-20240623-174819',
     },
-    whitelist: releaseCandidateHelloworldMatchingList,
+    // whitelist: releaseCandidateHelloworldMatchingList,
```
Comment (daniel-savu) [inline typescript/infra/config/environments/mainnet3/agent.ts:290]: should we add back this comment?
```
    // We're temporarily (ab)using the RC relayer as a way to increase
    // message throughput.
```

**Review by daniel-savu — APPROVED** [review]
lgtm!

File: typescript/infra/src/agents/index.ts (hunk 2 above)
Comment (tkporter) [reply typescript/infra/src/agents/index.ts:142]: will do

File: rust/Cargo.lock (hunk 1 above)
Comment (tkporter) [reply rust/Cargo.lock:7040]: Screen Shot 2024-06-24 at 2 24 42 PM

Whoa i made a typo! and we have this same typo in a bunch of other places that I've just fixed. Wild!

It shows up as a compilation warning but we have a lot of noise from some Solana warnings that it was ignored

Comment (tkporter) [reply rust/Cargo.lock:7040]: nice catch

**Pull request merged**